draft-ietf-rats-architecture.md

```diff
@@ -1188,64 +1188,80 @@ in terms of time(RX)-time(RG), then the Relying Party can check
+        |----Evidence------->|                           |              .
+        |    {time(EG)}   time(ER)--Evidence{time(EG)}-->|              .
+        |                    |                        time(RG)          .
+        |                 time(RA)<-Attestation Result---|              .
+        |                    |        {time(RX)}         |              .
+        ~                    ~                           ~              .
+        |                 time(HD')<------------------------------------/
+   time(HD')<-----------------------------------------------------------/
+        |                    |                        time(HD')<--------/
         |                 time(OP)                       |
 ~~~~
 
-In comparison with example 1, the time considerations in this example
-go into more detail with respect to the life-cycle of Claims and
-Evidence. While the goal is to create up-to-date and recent Evidence as soon as possible, typically there is a latency between value generation and Attester awareness.
-
-At time(AA) the Attesting Environment is able to trigger an event
-(e.g. based on an Event-Condition-Action model) to create attestation
-Evidence that is as recent as possible. In essence, at time(AA) the
-Attesting Environment is aware of new values that where generated at
-time(VG) and corresponding Claim values are collected immediately.
-Consecutively, Evidence based on relevant "old" Claims and the just
-collected "new" Claims is generated at time(EG). In essence, the Claims used to generate the Evidence are generated at various time(VG) before time(AA).
-
-In order to create attestation Evidence at
-at time(AA), the Attester requires a fresh (i.e. not expired)
-centrally generated handle that has been distributed to all involved
-entities.
-
-In general, The duration a handle remains fresh depends on
-the content-type of the handle. If it is a (relative or absolute)
-timestamp, clocks synchronized with a shared and trustworthy source of
-time are required. If another value type is used as a handle, the
-reception time of the handle time(HD) provides an epoch (relative time
-of zero) for measuring the duration of validity (similar to a
-heart-beat timeout). From the point of view of a Verifier, validity of
-Evidence is only given if the handle used in Evidence satisfies
-delta(time(HD),time(EG))distribution-interval.
-
-In this usage scenario, time(VG), time(AA), and time(EG) are tightly
-coupled. Also, the absolute point in time at which a handle is received
-by all three entities is assumed to be close to identical.
-
-## Example 4: Nonce-based Background-Check Model Example
+At time(AA), the Attesting Environment is able to trigger an event (e.g. based on an Event-Condition-Action model) to generate Evidence as soon/recent as possible at time (EG).
+This on-change generated Evidence can be based on a mix of relevant "old" Claim values and just recently generated and collected "new" Claim values.
```

Comment:
How is this sentence different from the other examples? (If it's not, then remove the sentence.)

Reply:
In practice today, it is not possible to include time(VG) as a value into evidence. Sometimes, the truth is that you can only derive time(AA). And time(VG) != time(AA). It gets even "worse" when you have no local clock and your "measure of time" is only "which handles did I already get?" All these scenarios are solvable and we do not want to include a complete problem statement here, just an example that readers faced with these kinds of challenges today can relate with.

I understand that you would like to subsume that context under time(EG) and I see how that reads simpler. I have to restate that I remain a bit worried, though, that this simplification does not reflect the real world,and could potentially lead to wrong interpretation of the security assumptions/consideration.